S/SDPLR/build_tarballs.jl

```diff
@@ -0,0 +1,44 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "SDPLR"
+version = v"1.0.3"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("http://sburer.github.io/files/SDPLR-1.03-beta.zip", "f1f945734f72e008fd7be8544b27341b179292c3304226563d9c0f6cf503b2eb")
+]
+
+# Bash recipe for building across all platforms
+# Even if the `Makefile` specifies `-o ../sdplr`,
+# On Windows with `libgfortran5`, `.exe` is added.
+# On Windows with `libgfortran3` or `libgfortran4`, nothing is added though.
+script = raw"""
+cd $WORKSPACE/srcdir/SDPLR*
+make LAPACK_LIB=-lopenblas BLAS_LIB=
+if [[ "${target}" == *-libgfortran5* ]]; then
+    EXT=${exeext}
+else
+    EXT=""
+fi
+install -Dvm 755 "sdplr${EXT}" "${bindir}/sdplr${exeext}"
```

Comment:
Just do
```suggestion
install -Dvm 755 sdplr* "${bindir}/sdplr${exeext}"
```
everywhere?

Reply:
Thanks, that did the trick, everything's green now :tada:

Reply:
Uhm, ok, not so fast:
```
[08:10:30]  ---> install -Dvm 755 sdplr* "${bindir}/sdplr${exeext}"
[08:10:30] created directory: '/workspace/destdir/bin'
[08:10:30] 'sdplr' -> '/workspace/destdir/bin/sdplr'
[08:10:30] 'sdplr.m' -> '/workspace/destdir/bin/sdplr'
[08:10:30] 'sdplr.params' -> '/workspace/destdir/bin/sdplr'
```
You see why verbose output is important?

Reply:
Indeed, we don't want to copy that

Reply:
Your solution was better, as it was more explicit

Reply:
If there was a way to ignore missing files, we could just do `install sdplr`  and `install sdplr$(exeext}`

Reply:
```sh
if [[ -f sdplr$(exeext} ]]; then
    # do something
else
    # do something else
fi
```
?

Reply:
Seems to work